extension/src/popup/views/SignTransaction/index.tsx

```diff
@@ -90,7 +90,7 @@ export const SignTransaction = () => {
   const transaction = SDK.TransactionBuilder.fromXDR(
     transactionXdr,
     networkPassphrase,
-  );
+  ) as any;
```

Comment:
I think I've run into this before, we can probably drop this `any` in the future if we refactor this to deal with the different transaction types separately.